challenge-13/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,199 @@
+
+// InitDB sets up a new SQLite database and creates the products table
+func InitDB(dbPath string) (*sql.DB, error) {
+	// TODO: Open a SQLite database connection
+	// TODO: Create the products table if it doesn't exist
+	// The table should have columns: id, name, price, quantity, category
+	db, err := sql.Open("sqlite3", dbPath)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := db.Ping(); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Ping must not be called. use (*database/sql.DB).PingContext

(noctx)